src/utils.jl

```diff
@@ -155,7 +145,7 @@ More convenient macro expansion, e.g.
     @expand @time foo()
 """
 macro expand(ex)
-  :(alias_gensyms(macroexpand($(ex,)[1])))
+  :(alias_gensyms(macroexpand($(@__MODULE__), $(ex,)[1])))
```

Comment:
Either splice in `$__module__`, or expand `@__MODULE__`, don't splice in `@__MODULE__`